test_unstructured_ingest/test-ingest-confluence2.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd "$SCRIPT_DIR"/.. || exit 1
+
+# Tests the scenario where --confluence-list-of-spaces has been provided, with
+# a big number of documents in the spaces.
+OUTPUT_FOLDER_NAME=confluence2
+OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
+
+if [ -z "$CONFLUENCE_USER_EMAIL" ] || [ -z "$CONFLUENCE_API_TOKEN" ]; then
+   echo "Skipping Confluence ingest test because the CONFLUENCE_USER_EMAIL or CONFLUENCE_API_TOKEN env var is not set."
+   exit 0
+fi
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+    --download-dir "$DOWNLOAD_DIR" \
+    --confluence-url https://unstructured-ingest-test.atlassian.net \
+    --confluence-user-email "$CONFLUENCE_USER_EMAIL" \
+    --confluence-api-token "$CONFLUENCE_API_TOKEN" \
+    --confluence-num-of-spaces 10 \
+    --confluence-list-of-spaces testteamsp1 \
+    --confluence-num-of-docs-from-each-space 250 \
```

Comment:
maybe instead we have this test use the defaults and validate it grabs everything?

Reply:
Agreed that we need to test that aspect (grabbing every doc). I now changed confluence1 (-diff) to use the default, and get all the docs.

In confluence2 (-large) I wanted to test if the scroll functionality can actually honor `--confluence-num-of-docs-from-each-space 250` rather than just going 3 scrolls and ingesting 300 docs, that's why I specifically mentioned 250 there

Reply:
What do you think?

Reply:
perfect!